### ISSUE TITLE
Error: statfs. no such file or directory

### ISSUE THREAD

**Issue by coresh** [mailcow/mailcow-dockerized#4063]
**Prior to placing the issue, please check following:** *(fill out each checkbox with an `X` once done)*
- [x] I understand that not following or deleting the below instructions will result in immediate closure and/or deletion of my issue.
- [x] I have understood that this bug report is dedicated for bugs, and not for support-related inquiries.
- [x] I have understood that answers are voluntary and community-driven, and not commercial support.
- [x] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

## Summary

Directories -> should be pre-created.
```
mkdir -p ./data/hooks/dovecot
mkdir -p ./data/hooks/phpfpm
mkdir -p ./data/hooks/postfix
mkdir -p ./data/hooks/rspamd
mkdir -p ./data/hooks/unbound
mkdir -p ./data/web/.well-known/acme-challenge
```
One-liner:
```
mkdir -p ./data/hooks/dovecot ./data/hooks/phpfpm ./data/hooks/postfix ./data/hooks/rspamd ./data/hooks/unbound ./data/web/.well-known/acme-challenge
```
### The reason:

1. To excluding `Logs` output
2. Successful buld images / containers. (`Else -> provides: stop build process`)

## Logs

```
Error: statfs ./data/hooks/dovecot: no such file or directory
Error: statfs ./data/hooks/phpfpm: no such file or directory
Error: statfs ./data/hooks/postfix: no such file or directory
Error: statfs ./data/hooks/rspamd: no such file or directory
Error: statfs ./data/hooks/unbound: no such file or directory
Error: statfs ./data/web/.well-known/acme-challenge: no such file or directory

```

## Reproduction

```
sudo docker-compose up -d
```

## System information

| Question | Answer |
| --- | --- |
| My operating system | Linux n-000001 5.11.13-200.fc33.x86_64 #1 SMP Sun Apr 11 02:08:26 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Is Apparmor, SELinux or similar active? |SELinux status:                 disabled |
| Virtualization technlogy (KVM, VMware, Xen, etc - **LXC and OpenVZ are not supported** | - |
| Server/VM specifications (Memory, CPU Cores) | 64Gb / 32 CPU Cores |
| Docker Version (`docker version`) | podman version 3.1.0  |
| Docker-Compose Version (`docker-compose version`) | podman-composer version  0.1.7dev|
| Reverse proxy (custom solution) | - |

- Output of `git diff origin/master`, any other changes to the code? If so, **please post them**.
- All third-party firewalls and custom iptables rules are unsupported. *Please check the Docker docs about how to use Docker with your own ruleset*. Nevertheless, iptabels output can help us to help you: `iptables -L -vn`, `ip6tables -L -vn`, `iptables -L -vn -t nat` and `ip6tables -L -vn -t nat`.
- DNS problems? Please run `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @172.22.1.254` (set the IP accordingly, if you changed the internal mailcow network) and post the output.

**Comment by andryyy** [comment]
Podman is probably still problematic. :)

**Comment by coresh** [comment]
> odman is probably still problematic. :)

Correct.

But,

The progress with the podman started from:

```
mkdir -p ./data/hooks/dovecot ./data/hooks/phpfpm ./data/hooks/postfix ./data/hooks/rspamd ./data/hooks/unbound ./data/web/.well-known/acme-challenge
```
Thank you

**Comment by andryyy** [comment]
Indeed!

I will add a .gitkeep to the folders. :)

**Comment by coresh** [comment]
> Indeed!
> 
> I will add a .gitkeep to the folders. :)

[Fixed by the commit: `891213c59c0f92cef79b3bfa18ed450fb628e6c6` &#187; ](https://github.com/mailcow/mailcow-dockerized/commit/891213c59c0f92cef79b3bfa18ed450fb628e6c6)

Thank you
